doc_src/cmds/set_color.rst

```diff
@@ -52,8 +55,8 @@ The following options are available:
 **-r** or **--reverse**
     Sets reverse mode.
 
-**-u** or **--underline**
-    Sets underlined mode.
+**-u** or **--underline** or **--curly-underline**
+    Set the underline mode.
```

Comment:
actually it seems better to give it an optional arg.
Then it's more obvious that we can't combine two underline shapes.

	-u --underline, --underline=curly, --underline=double
	-u[STYLE] --underline[=STYLE] where STYLE is `single` (default), `curly` or `double`

could also call it undercurl but that doesn't work with this approach